### ISSUE TITLE
claat: support user-input in command-line snippets

### ISSUE THREAD

**Issue by simos** [googlecodelabs/tools#24]
The command-line snippet can be enhanced in order to show which is the user-input text among the text output of the commands.

Existing codelabs are using different conventions in showing command-line commands. Some are omitting the shell prompt altogether, others do not show any output from the commands which can be confusing to the users. 

For a codelabs author, they should just make bold the parts of the command-line input that is actual user-input. Then claat should show this text in a distinctive way to denote user-input.

See [http://i.imgur.com/yM0PQ63.png](screenshot of commands with user-input in bold)

**Comment by x1ddos** [comment]
Yeah, many codelabs are not uniform. The disadvantage of this is that a hypothetical click-to-copy button would be useless. We were trying to encourage authors to provide snippets without the a shell prompt.

Not sure about pros and cons though.
